.github/workflows/backwards_compatibility_tests_workflow.yml

```diff
@@ -13,9 +13,9 @@ jobs:
   Restart-Upgrade-BWCTests-NeuralSearch:
     strategy:
       matrix:
-        java: [21, 23]
-        os: [ubuntu-latest,windows-latest]
```

Comment:
Does OS core skips a windows platform run?

Reply:
I'm not sure how to check this. What that matter to us, we are not releasing any platform dependent artifacts

Reply:
I personally think it's more important to check with different JDK versions (21 and 23) because java language features can be different, recently some PRs has problem with that. With jdk 23 added it will be 2x of actions, so our heavy CI will be even more heavier